VoodooI2CHID/VoodooI2CTouchscreenHIDEventDriver.cpp

```diff
@@ -25,67 +25,52 @@ OSDefineMetaClassAndStructors(VoodooI2CTouchscreenHIDEventDriver, VoodooI2CMulti
 // Override of VoodooI2CMultitouchHIDEventDriver
 
 bool VoodooI2CTouchscreenHIDEventDriver::checkFingerTouch(AbsoluteTime timestamp, VoodooI2CMultitouchEvent event) {
-    bool got_transducer = false;
+    absolutetime_to_nanoseconds(timestamp, &last_interaction);
     
-    // If there is a finger touch event, decide if it is single or multitouch.
+    // Avoid phantom clicks happening after scrolling
+    if ((last_interaction - last_multitouch_interaction) < MULTITOUCH_TIMEOUT) {
+        return false;
+    }
```

Comment:
I return `false` here if the touch happens too quickly after a scroll, which prevents phantom clicks.

However, returning `false` here has the side effect that `multitouch_interface->handleInterruptReport()` will be called. Is that what should happen, or should we return `true` and stop it from being called?